pkg/operator/render.go

```diff
@@ -196,17 +196,23 @@ func clusterDNSIP(iprange string) (string, error) {
 }
 
 func ipFamilies(serviceCIDRs []string) (mcfgv1.IPFamiliesType, error) {
-	var ipv4, ipv6 bool
-	for _, cidr := range serviceCIDRs {
+	var ipv4, ipv6, v6primary bool
+	for i, cidr := range serviceCIDRs {
 		if utilnet.IsIPv6CIDRString(cidr) {
 			ipv6 = true
+			if i == 0 {
```

Comment:
So essentially the ordering is the only thing that matters when determining the primary, right?

Reply:
Right. Whatever IP version is first in the list must be primary. We've been forcing that to be ipv4 at install-time, but after this change we can stop doing that.